src/components/layout-components/localization-buttons.tsx

```diff
@@ -13,34 +14,29 @@ const LocalizationButtons = () => {
   /**
    * Method to handle locale change
    */
-  const handleLocaleChange = (
-    event: React.MouseEvent<HTMLElement>,
-    newLanguage: Language | null
-  ) => {
+  const handleLocaleChange = (_event: MouseEvent<HTMLElement>, newLanguage: Language | null) => {
```

Comment:
Is there a reason newLanguage is typed as Language | null?
If so, it might be a good idea to check if the newLanguage is null and handle appropriately rather than end up setting the localization to null.

Reply:
Removed null and added null check.